pkg/contracts/src/ProxyOwnableUpgrader.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.19;
+
+import {OwnableUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/access/OwnableUpgradeable.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+
+contract ProxyOwnableUpgrader is OwnableUpgradeable, UUPSUpgradeable {
+    error CallerNotOwner(address _caller, address _owner);
+
+    function initialize(address initialOwner) public initializer {
+        _transferOwnership(initialOwner);
+    }
```

Comment:
must call initialize

Reply:
initialize is just transfering to msg.sender no ?

Reply:
Not needed

Reply:
The initializer does the set initialized to true:
![Uploading image.png…]()